app/assets/stylesheets/components/_footer.scss

```diff
@@ -0,0 +1,9 @@
+.footer {
+  // background-color: $white-button;
+  position: absolute;
+  width: 44px;
+  height: 44px;
+  left: 41px;
+  top: 743px;
+  border-radius: 50px;
+}
```

Comment:
pay attention to the end of files. Github will ask you to add one extra line.

Reply:
Oh ok I see. I will add an extra line today. Thanks!